src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The class Rectangle should not start with an empty line. Please remove the empty line at the beginning of the class definition.